pkg/operator/resource/resourcecread/apps.go

```diff
@@ -0,0 +1,34 @@
+package resourceread
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+)
+
+var (
+	appsScheme = runtime.NewScheme()
+	appsCodecs = serializer.NewCodecFactory(appsScheme)
+)
+
+func init() {
+	if err := appsv1.AddToScheme(appsScheme); err != nil {
+		panic(err)
+	}
+}
+
+func ReadDeploymentV1OrDie(objBytes []byte) *appsv1.Deployment {
```

Comment:
@deads2k as this is library, i would like to have godoc on these ;-) i'm fine with follow up after we move everything we need, not going to hold this on godoc...